src/imgtools/dicom/input/sr_utils.py

```
@@ -0,0 +1,94 @@
+
+    series_uids = set()
+    sop_uids = set()
+
+    for evidence_seq in sr.CurrentRequestedProcedureEvidenceSequence:
+        if not "ReferencedSeriesSequence" in evidence_seq:
+            continue
+
+        for series_seq in evidence_seq.ReferencedSeriesSequence:
+            series_uids.add(series_seq.SeriesInstanceUID)
+
+            if not "ReferencedSOPSequence" in series_seq:
```

Ruff (0.8.2)

Test for membership should be `not in`

Convert to `not in`

(E713)